### ISSUE TITLE
Terraform crash since upgrading to version 011.3

### ISSUE THREAD

**Issue by dfrappart** [hashicorp/terraform#20921]
error when launching any terraform cli command since upgrade to terraform 0.11.3 on Windows.
Works still well on linux though.
Below is crashlog content: 

2019/04/03 17:42:42 [INFO] Terraform version: 0.11.13  
2019/04/03 17:42:42 [INFO] Go runtime version: go1.11.5
2019/04/03 17:42:42 [INFO] CLI args: []string{"C:\\Program Files (x86)\\Terraform\\terraform.exe", "init", "-upgrade=true"}
2019/04/03 17:42:42 [DEBUG] Attempting to open CLI config file: C:\Users\David\AppData\Roaming\terraform.rc
2019/04/03 17:42:42 Loading CLI configuration from C:\Users\David\AppData\Roaming\terraform.rc
2019/04/03 17:42:42 [INFO] CLI command args: []string{"init", "-upgrade=true"}
2019/04/03 17:42:42 [DEBUG] command: loading backend config file: D:\Users\David\Docs\IaC\terraform\Azure\AKS Lab
2019/04/03 17:42:42 [TRACE] module source: "github.com/dfrappart/Terra-AZModuletest//Modules//01 ResourceGroup/"
2019/04/03 17:42:42 [TRACE] detected module source "git::https://github.com/dfrappart/Terra-AZModuletest.git"
2019/04/03 17:42:42 [DEBUG] fetching module from git::https://github.com/dfrappart/Terra-AZModuletest.git
2019/04/03 17:42:42 [DEBUG] fetching "git::https://github.com/dfrappart/Terra-AZModuletest.git" with key "1.ResourceGroupHubSpoke;github.com/dfrappart/Terra-AZModuletest//Modules//01 ResourceGroup/"
2019/04/03 17:42:45 [DEBUG] found "git::https://github.com/dfrappart/Terra-AZModuletest.git" in ".terraform\\modules\\0b99928a89d73b5c2714c9cfdb2355e0": true
2019/04/03 17:42:45 [TRACE] "git::https://github.com/dfrappart/Terra-AZModuletest.git" stored in ".terraform\\modules\\0b99928a89d73b5c2714c9cfdb2355e0"
2019/04/03 17:42:45 [DEBUG] plugin: waiting for all plugin processes to complete...
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/hashicorp/terraform/config/module.Storage.loadManifest(0x15638ee0, 0x12, 0x2a1cba0, 0x156c0120, 0x2, 0x1540f628, 0xffcfffdf, 0x1e7ffef, 0x29f0251, 0x7, ...)
	/opt/gopath/src/github.com/hashicorp/terraform/config/module/storage.go:110 +0x33b
github.com/hashicorp/terraform/config/module.Storage.recordModule(0x15638ee0, 0x12, 0x2a1cba0, 0x156c0120, 0x2, 0x1540f628, 0x15496b41, 0x43, 0x1550c5a0, 0x5b, ...)
	/opt/gopath/src/github.com/hashicorp/terraform/config/module/storage.go:127 +0x36
github.com/hashicorp/terraform/config/module.(*Tree).getChildren(0x15772190, 0x1540b000, 0x157721ac, 0x1540b000, 0x9abf8f)
	/opt/gopath/src/github.com/hashicorp/terraform/config/module/tree.go:330 +0x7fe
github.com/hashicorp/terraform/config/module.(*Tree).Load(0x15772190, 0x1540b000, 0x0, 0x0)
	/opt/gopath/src/github.com/hashicorp/terraform/config/module/tree.go:178 +0x78
github.com/hashicorp/terraform/command.getModules(0x1546bc00, 0x1568f040, 0x2f, 0x157bf902, 0x20, 0x15826870)
	/opt/gopath/src/github.com/hashicorp/terraform/command/get.go:84 +0xf3
github.com/hashicorp/terraform/command.(*InitCommand).Run(0x1546bc00, 0x15471910, 0x1, 0x2, 0x156a6108)
	/opt/gopath/src/github.com/hashicorp/terraform/command/init.go:179 +0x947
github.com/mitchellh/cli.(*CLI).Run(0x154d4280, 0x154d4280, 0x15595e9c, 0x1)
	/opt/gopath/pkg/mod/github.com/mitchellh/cli@v0.0.0-20171129193617-33edc47170b5/cli.go:255 +0x1ba
main.wrappedMain(0x0)
	/opt/gopath/src/github.com/hashicorp/terraform/main.go:223 +0xa15
main.realMain(0x0)
	/opt/gopath/src/github.com/hashicorp/terraform/main.go:100 +0x7f
main.main()
	/opt/gopath/src/github.com/hashicorp/terraform/main.go:36 +0x2e

**Comment by AErmie** [comment]
I encountered the same crash. Here is my crashlog content:

2019/04/04 08:10:15 [INFO] Terraform version: 0.11.13  
2019/04/04 08:10:15 [INFO] Go runtime version: go1.11.5
2019/04/04 08:10:15 [INFO] CLI args: []string{"C:\\Terraform\\terraform.exe", "init", "-backend-config=key=corecac"}
2019/04/04 08:10:15 [DEBUG] Attempting to open CLI config file: C:\Users\adine\AppData\Roaming\terraform.rc
2019/04/04 08:10:15 [DEBUG] File doesn't exist, but doesn't need to. Ignoring.
2019/04/04 08:10:15 [INFO] CLI command args: []string{"init", "-backend-config=key=corecac"}
2019/04/04 08:10:15 [DEBUG] command: loading backend config file: C:\Users\adine\Documents\GitHub\terraformmvptest\networking\deployments\core
2019/04/04 08:10:15 [TRACE] module source: "../../vnets/core"
2019/04/04 08:10:15 [DEBUG] plugin: waiting for all plugin processes to complete...
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/hashicorp/terraform/config/module.Storage.loadManifest(0xc0002fdce0, 0x12, 0x33c4180, 0xc0003c5fc0, 0x1, 0xc0002cd6b0, 0xc0002fdd60, 0x6020000, 0x0, 0x1b, ...)
	/opt/gopath/src/github.com/hashicorp/terraform/config/module/storage.go:110 +0x487
github.com/hashicorp/terraform/config/module.Storage.moduleDir(0xc0002fdce0, 0x12, 0x33c4180, 0xc0003c5fc0, 0x1, 0xc0002cd6b0, 0xc0002fdd80, 0x1d, 0x2, 0x2, ...)
	/opt/gopath/src/github.com/hashicorp/terraform/config/module/storage.go:185 +0x64
github.com/hashicorp/terraform/config/module.Storage.findModule(0xc0002fdce0, 0x12, 0x33c4180, 0xc0003c5fc0, 0x1, 0xc0002cd6b0, 0xc0002fdd80, 0x1d, 0x0, 0x0, ...)
	/opt/gopath/src/github.com/hashicorp/terraform/config/module/storage.go:258 +0x91
github.com/hashicorp/terraform/config/module.(*Tree).getChildren(0xc000336a00, 0xc0002daea0, 0xc000336a38, 0xb28719, 0x300d660)
	/opt/gopath/src/github.com/hashicorp/terraform/config/module/tree.go:234 +0x155e
github.com/hashicorp/terraform/config/module.(*Tree).Load(0xc000336a00, 0xc0002daea0, 0x0, 0x0)
	/opt/gopath/src/github.com/hashicorp/terraform/config/module/tree.go:178 +0xa5
github.com/hashicorp/terraform/command.getModules(0xc0003e8ea0, 0xc00030cb40, 0x4c, 0xc0002f6301, 0x23, 0xc0002f6270)
	/opt/gopath/src/github.com/hashicorp/terraform/command/get.go:84 +0x127
github.com/hashicorp/terraform/command.(*InitCommand).Run(0xc0003e8ea0, 0xc00006c120, 0x1, 0x2, 0xc0002e7750)
	/opt/gopath/src/github.com/hashicorp/terraform/command/init.go:179 +0xac7
github.com/mitchellh/cli.(*CLI).Run(0xc0003d8140, 0xc0003d8140, 0xc0001bbd20, 0x1)
	/opt/gopath/pkg/mod/github.com/mitchellh/cli@v0.0.0-20171129193617-33edc47170b5/cli.go:255 +0x20e
main.wrappedMain(0x0)
	/opt/gopath/src/github.com/hashicorp/terraform/main.go:223 +0xb55
main.realMain(0x0)
	/opt/gopath/src/github.com/hashicorp/terraform/main.go:100 +0xbe
main.main()
	/opt/gopath/src/github.com/hashicorp/terraform/main.go:36 +0x35

**Comment by sinahwz** [comment]
Getting the same error on Windows, when running any commands on some environments. 

- Windows 
- terraform --version >>> Terraform v0.11.13

**Comment by divgo** [comment]
Getting this error as well. Running on Windows 10 and trying to INIT a configuration that uses Modules. Same Configuration worked fine with 0.11.12

```
2019/04/10 10:22:25 [INFO] Terraform version: 0.11.13  
2019/04/10 10:22:25 [INFO] Go runtime version: go1.11.5
2019/04/10 10:22:25 [INFO] CLI args: []string{"C:\\ashley\\tools\\terraform.exe", "init", "--backend-config=access_key=cFLMQVTg8PztOll1NGcQukiwr8ouP4VLtU/L+dY7I2jCsCIkin5prrU1PMNjhm0mow0B6wLTDJ43gnQAWX/6yw=="}
2019/04/10 10:22:25 [DEBUG] Attempting to open CLI config file: C:\Users\SMcEnery\AppData\Roaming\terraform.rc
2019/04/10 10:22:25 [DEBUG] File doesn't exist, but doesn't need to. Ignoring.
2019/04/10 10:22:25 [INFO] CLI command args: []string{"init", "--backend-config=access_key=cFLMQVTg8PztOll1NGcQukiwr8ouP4VLtU/L+dY7I2jCsCIkin5prrU1PMNjhm0mow0B6wLTDJ43gnQAWX/6yw=="}
2019/04/10 10:22:25 [DEBUG] command: loading backend config file: C:\ashley\scm\terraform-azure\agr-stage
2019/04/10 10:22:25 [TRACE] module source: "agr-stage"
2019/04/10 10:22:25 [DEBUG] plugin: waiting for all plugin processes to complete...
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/hashicorp/terraform/config/module.Storage.loadManifest(0xc00040ae00, 0x12, 0x33c4180, 0xc0003dc000, 0x1, 0xc000269f20, 0xc00040ae80, 0x5ab06a0, 0x0, 0x13, ...)
	/opt/gopath/src/github.com/hashicorp/terraform/config/module/storage.go:110 +0x487
github.com/hashicorp/terraform/config/module.Storage.moduleDir(0xc00040ae00, 0x12, 0x33c4180, 0xc0003dc000, 0x1, 0xc000269f20, 0xc00040aea0, 0x15, 0x2, 0x2, ...)
	/opt/gopath/src/github.com/hashicorp/terraform/config/module/storage.go:185 +0x64
github.com/hashicorp/terraform/config/module.Storage.findModule(0xc00040ae00, 0x12, 0x33c4180, 0xc0003dc000, 0x1, 0xc000269f20, 0xc00040aea0, 0x15, 0x0, 0x0, ...)
	/opt/gopath/src/github.com/hashicorp/terraform/config/module/storage.go:258 +0x91
github.com/hashicorp/terraform/config/module.(*Tree).getChildren(0xc000426600, 0xc00037a8a0, 0xc000426638, 0xb28719, 0x300d660)
	/opt/gopath/src/github.com/hashicorp/terraform/config/module/tree.go:234 +0x155e
github.com/hashicorp/terraform/config/module.(*Tree).Load(0xc000426600, 0xc00037a8a0, 0x0, 0x0)
	/opt/gopath/src/github.com/hashicorp/terraform/config/module/tree.go:178 +0xa5
github.com/hashicorp/terraform/command.getModules(0xc0003cf380, 0xc0003fa0c0, 0x27, 0xc000036801, 0x23, 0xc000036810)
	/opt/gopath/src/github.com/hashicorp/terraform/command/get.go:84 +0x127
github.com/hashicorp/terraform/command.(*InitCommand).Run(0xc0003cf380, 0xc000046160, 0x1, 0x2, 0xc0002cb770)
	/opt/gopath/src/github.com/hashicorp/terraform/command/init.go:179 +0xac7
github.com/mitchellh/cli.(*CLI).Run(0xc0003be140, 0xc0003be140, 0xc0001b9d20, 0x1)
	/opt/gopath/pkg/mod/github.com/mitchellh/cli@v0.0.0-20171129193617-33edc47170b5/cli.go:255 +0x20e
main.wrappedMain(0x0)
	/opt/gopath/src/github.com/hashicorp/terraform/main.go:223 +0xb55
main.realMain(0x0)
	/opt/gopath/src/github.com/hashicorp/terraform/main.go:100 +0xbe
main.main()
	/opt/gopath/src/github.com/hashicorp/terraform/main.go:36 +0x35
```

**Comment by divgo** [comment]
> Getting this error as well. Running on Windows 10 and trying to INIT a configuration that uses Modules. Same Configuration worked fine with 0.11.12
> 
> ```
> 2019/04/10 10:22:25 [INFO] Terraform version: 0.11.13  
> 2019/04/10 10:22:25 [INFO] Go runtime version: go1.11.5
> 2019/04/10 10:22:25 [INFO] CLI args: []string{"C:\\ashley\\tools\\terraform.exe", "init", "--backend-config=access_key=cFLMQVTg8PztOll1NGcQukiwr8ouP4VLtU/L+dY7I2jCsCIkin5prrU1PMNjhm0mow0B6wLTDJ43gnQAWX/6yw=="}
> 2019/04/10 10:22:25 [DEBUG] Attempting to open CLI config file: C:\Users\SMcEnery\AppData\Roaming\terraform.rc
> 2019/04/10 10:22:25 [DEBUG] File doesn't exist, but doesn't need to. Ignoring.
> 2019/04/10 10:22:25 [INFO] CLI command args: []string{"init", "--backend-config=access_key={{AZURE ACCESS KEY}}"}
> 2019/04/10 10:22:25 [DEBUG] command: loading backend config file: C:\ashley\scm\terraform-azure\agr-stage
> 2019/04/10 10:22:25 [TRACE] module source: "agr-stage"
> 2019/04/10 10:22:25 [DEBUG] plugin: waiting for all plugin processes to complete...
> panic: runtime error: index out of range
> 
> goroutine 1 [running]:
> github.com/hashicorp/terraform/config/module.Storage.loadManifest(0xc00040ae00, 0x12, 0x33c4180, 0xc0003dc000, 0x1, 0xc000269f20, 0xc00040ae80, 0x5ab06a0, 0x0, 0x13, ...)
> 	/opt/gopath/src/github.com/hashicorp/terraform/config/module/storage.go:110 +0x487
> github.com/hashicorp/terraform/config/module.Storage.moduleDir(0xc00040ae00, 0x12, 0x33c4180, 0xc0003dc000, 0x1, 0xc000269f20, 0xc00040aea0, 0x15, 0x2, 0x2, ...)
> 	/opt/gopath/src/github.com/hashicorp/terraform/config/module/storage.go:185 +0x64
> github.com/hashicorp/terraform/config/module.Storage.findModule(0xc00040ae00, 0x12, 0x33c4180, 0xc0003dc000, 0x1, 0xc000269f20, 0xc00040aea0, 0x15, 0x0, 0x0, ...)
> 	/opt/gopath/src/github.com/hashicorp/terraform/config/module/storage.go:258 +0x91
> github.com/hashicorp/terraform/config/module.(*Tree).getChildren(0xc000426600, 0xc00037a8a0, 0xc000426638, 0xb28719, 0x300d660)
> 	/opt/gopath/src/github.com/hashicorp/terraform/config/module/tree.go:234 +0x155e
> github.com/hashicorp/terraform/config/module.(*Tree).Load(0xc000426600, 0xc00037a8a0, 0x0, 0x0)
> 	/opt/gopath/src/github.com/hashicorp/terraform/config/module/tree.go:178 +0xa5
> github.com/hashicorp/terraform/command.getModules(0xc0003cf380, 0xc0003fa0c0, 0x27, 0xc000036801, 0x23, 0xc000036810)
> 	/opt/gopath/src/github.com/hashicorp/terraform/command/get.go:84 +0x127
> github.com/hashicorp/terraform/command.(*InitCommand).Run(0xc0003cf380, 0xc000046160, 0x1, 0x2, 0xc0002cb770)
> 	/opt/gopath/src/github.com/hashicorp/terraform/command/init.go:179 +0xac7
> github.com/mitchellh/cli.(*CLI).Run(0xc0003be140, 0xc0003be140, 0xc0001b9d20, 0x1)
> 	/opt/gopath/pkg/mod/github.com/mitchellh/cli@v0.0.0-20171129193617-33edc47170b5/cli.go:255 +0x20e
> main.wrappedMain(0x0)
> 	/opt/gopath/src/github.com/hashicorp/terraform/main.go:223 +0xb55
> main.realMain(0x0)
> 	/opt/gopath/src/github.com/hashicorp/terraform/main.go:100 +0xbe
> main.main()
> 	/opt/gopath/src/github.com/hashicorp/terraform/main.go:36 +0x35
> ```

Found that if I delete the `.terraform` folder and run INIT again, it works.

**Comment by mkv2019** [comment]
Getting similar error as well. 

`rraform plan -out=terraform.nld2-pipeline-int.plan -refresh=true -module-depth=1 -input=false  -var 'ssh_public_key_contents=ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDYHcmxVsj16qLjSjiN+nT+RpaC6JYi3lJfI+XUqFi356Suz8RhHVJCEnfYplNVVYlhYzzamqZZTZ74phyuUp+0k23Vf7F4IG4Obw7m/aFOAPyM0SWhIK0ihOo7wCd7c+rs0Tz7zV/TNDmrkC2bziKMMckQPscr2HSLPv3chw4yJu5C+km278CxCwyKpz0KV9Andjlvln/cMbxadFPWg9tt1mA13IODF5JzXmfU3R2kM1ydvF/aIapyzI+6Xar1Qh1SM5BrJIdpgI9U0toE8lvePkpxkkKM1tq+zlSInqgO/dmgQR09fBZxmm7WuepVXCnTncU6UeDU7CZOtFqoEOjAPzFkhiM2wZxSeyVc5qxVHr2ttMSCDXq13xOGXGtIh9F9ijCvuiQMCvQQYohwcUA8MpdZKLmTJX6o2dR15XvGYEgt56E55wo6TM0sjicvJrBRRfPknE5jiNv/aoqO2pGz4T7oflnfKVcNWrC3iYF7/cUsHoqeNWJsBgO167jd/pdklMegCh8qosiE3GaRouXLNCBmUQv+c9iYEjcqSr6bPR4wrVts0KkKqQsxVJ1rJdwXB8KDNDZz7JZqzcLdmeMyPEhFcQg36FVuPhm5dMIcp2qq+GnaNL/tGhnj9hjIsEs0p30izJkthbcWdbDUIDGpMSq8BTUjAda7dBkOSId9JQ== uedanger'  -var 'cluster=nld2-pipeline-int'  -state=terraform.nld2-pipeline-int.tfstate
- module.azure_admin_collection
  Updating source "../../modules/azure_admin_collection"
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/hashicorp/terraform/config/module.Storage.loadManifest(0xc000430560, 0x12, 0x33c4180, 0xc000402040, 0x2, 0xc00023e2e0, 0xc000465310, 0x477f2f, 0xc000465310, 0x42cb08, ...)
        /opt/gopath/src/github.com/hashicorp/terraform/config/module/storage.go:110 +0x487
github.com/hashicorp/terraform/config/module.Storage.recordModule(0xc000430560, 0x12, 0x33c4180, 0xc000402040, 0x2, 0xc00023e2e0, 0xc00041e391, 0x24, 0xc0004829c0, 0x3d, ...)
        /opt/gopath/src/github.com/hashicorp/terraform/config/module/storage.go:127 +0x64
github.com/hashicorp/terraform/config/module.(*Tree).getChildren(0xc000492d80, 0xc0002f6b10, 0xc000492db8, 0xb28719, 0x300d660)
        /opt/gopath/src/github.com/hashicorp/terraform/config/module/tree.go:330 +0x9d2
github.com/hashicorp/terraform/config/module.(*Tree).Load(0xc000492d80, 0xc0002f6b10, 0x0, 0x0)
        /opt/gopath/src/github.com/hashicorp/terraform/config/module/tree.go:178 +0xa5
github.com/hashicorp/terraform/command.getModules(0xc0003fd040, 0xc00034ed80, 0x38, 0xc00034ed02, 0x38, 0x0)
        /opt/gopath/src/github.com/hashicorp/terraform/command/get.go:84 +0x127
github.com/hashicorp/terraform/command.(*GetCommand).Run(0xc0003fd040, 0xc0000840e0, 0x1, 0x2, 0xc0002f9710)
        /opt/gopath/src/github.com/hashicorp/terraform/command/get.go:43 +0x203
github.com/mitchellh/cli.(*CLI).Run(0xc0003ea140, 0xc0003ea140, 0xc0001d5d20, 0x1)
        /opt/gopath/pkg/mod/github.com/mitchellh/cli@v0.0.0-20171129193617-33edc47170b5/cli.go:255 +0x20e
main.wrappedMain(0x0)
        /opt/gopath/src/github.com/hashicorp/terraform/main.go:223 +0xb55
main.realMain(0x0)
        /opt/gopath/src/github.com/hashicorp/terraform/main.go:100 +0xbe
main.main()
        /opt/gopath/src/github.com/hashicorp/terraform/main.go:36 +0x35



!!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
`

**Comment by theoc31** [comment]
Worked for me @divgo too - nice one!

**Comment by teamterraform** [comment]
Hi all! Sorry for this crash and thanks for reporting it.

There was a bug in a file format upgrade codepath in Terraform 0.11.13 that was trying to update a legacy file format in the `.terraform` directory, but crashing on some input. That bug was subsequently fixed in 0.11.14, so we're going to close this out now. Thanks again!
